Add Cybersecurity Best Practices

sgrest opened the pull request:
New cybersecurity contents are added to this page to provide the clients with security measures that they need to apply or implement when installing or developing using Bonita.

sgrest committed:
Add Cybersecurity Best Practices

New cybersecurity contents are added to this page to provide the clients with security measures that they need to apply or implement when installing or developing using Bonita.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/bonitasoft/bonita-doc?pullRequest=456) <br/>All committers have signed the CLA.

educhastenier commented:
test this please

educhastenier committed:
Fixed formatting

educhastenier commented:
retest this please

educhastenier committed:
fix formatting

educhastenier commented:
retest this please

educhastenier committed:
fix small formulating words

educhastenier commented:
retest this please

educhastenier commented on md/first-steps-after-setup.md:
```diff
@@ -12,11 +12,43 @@ In the URL field of your web browser, enter the local host address, e.g. `http:/
+In Bonita, connectors enable connections to your local information system or online services. The authentication credentials used for these connections are of great value to the business and as a result, they need to be protected. It is highly recommended to choose strong, random and long passwords, as explained above, to makes it difficult or even improbable for one to guess the password. 
+
+#### Prohibit or limit the usage of Java Client
+The Java Client is a tool that can be downloaded and installed for configuration purposes by the administrators. If such tool is of no use for your application, it is highly recommended to prohibit the usage of Java Client in your deployment environment. Otherwise, its usage must be limited to administrators using a local connection or authorized IP addresses.
```
What do you mean with Java Client? How can you prohibit its usage?
Don't you rather mean to deactivate the Engine HTTP API?

sgrest replied:
Thanks for your comment. I didn't know that  Engine HTTP API can be deactivated in configuration files. I'll change and reformulate this part.

tbouffard replied:
See https://documentation.bonitasoft.com/bonita/7.3/configure-client-of-bonita-bpm-engine#toc5 for server side configuration

sgrest committed:
Updated the java client section

I made the required changes. I had to stay very simple without giving too much details. Please let me know if the sentences in this section are clear.

tbouffard commented:
@sgrest sorry for the so late review, I haven't seen your new commit. This looks good to me now
It's seem to me that this also applies to the 7.3 version, I am checking this right now

tbouffard commented:
I have just review the content of this PR and all pages reference here exist in 7.3.
So I am rebasing the PR on the 7.3 branch

tbouffard commented:
Finally, too much changes between 7.3 and 7.4, so I am merging this PR in the 7.4 branch
